web-ng/src/app/components/select-dialog/select-dialog.component.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Inject } from '@angular/core';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+
+export interface DialogData {
+  clickedPolygons: google.maps.Polygon[];
```

Comment:
Maybe make this more generic, eg "features", and accept Feature instead of google.maps.Polygon?

Reply:
Agreed with being generic. At this point, it's hard to get the feature itself. So I created a `FeatureData` in the dialog component and put all the data in there.